lte/protos/session_manager.proto

```diff
@@ -1009,12 +1005,15 @@ service SetInterfaceForUserPlane {
     rpc SendPagingReuest(UPFPagingInfo) returns (magma.orc8r.Void) {}
 }
 
+//SET API for 3 procedures request from AMF to SMF
+service AmfPduSessionSmContext {
+  rpc SetAmfSessionContext(SetSMSessionContext) returns (SmContextVoid) {}
```

Comment:
Hi @Chandhu-Wavelabs ! Do you mind adding a one sentence summary above for both `SetAmfSessionContext` and `SetSmfNotification`?

Reply:
> Hi @Chandhu-Wavelabs ! Do you mind adding a one sentence summary above for both `SetAmfSessionContext` and `SetSmfNotification`?

Hi @themarwhal, added comments for these two methods